src/pages/index.js

```diff
@@ -14,12 +14,14 @@ export default function IndexPage({ data }) {
       <SEO />
       <h2>All posts</h2>
       <ul>
-        {data.posts.edges.map(({ node }) => (
-          <li key={node.fields.slug}>
-            <Link to={node.fields.slug}>{node.frontmatter.title}</Link>
-            {` - ${node.fields.dateWithYear}`}
-          </li>
-        ))}
+        {data.posts.edges
+          .filter(({ node }) => !node.frontmatter.rss_only)
```

Comment:
This will work just as effectively as filtering at the GraphQL query layer (because it will be SSR/statically rendered) but I usually _prefer_ to do it at the GraphQL layer, like so:

```graphql
query IndexQuery {
  posts: allMarkdownRemark(
    sort: { fields: [fields___date], order: DESC }
    filter: { frontmatter: { rss_only: { ne: true }}}
  ) {
    # the other stuff
  }
}
```

Reply:
Also there are only a few rss_only posts right? The `totalCount` went from 44 to 41 with this filter applied.